### PR TITLE
Feature: Added snack bar

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added snack bar component.

Live Link: https://sleepy-wescoff-c77267.netlify.app/

### CODE REVIEW

**Review by syed-shaida-hussain — APPROVED** [review]
Code looks very clean. No changes needed.

**Comment by poojakelkar** [comment]
Thanks

File: components/snackbar/styles.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+:root{
```
Comment (saifudinbadani) [inline components/snackbar/styles.css:1]: make this a seperate file, it will be easy to edit because these colours will be used in every component.

**Review by saifudinbadani — APPROVED** [review]
Otherwise, looks good to merge.

File: components/snackbar/styles.css (hunk 1 above)
Comment (poojakelkar) [reply components/snackbar/styles.css:1]: Thanks @saifudinbadani . I have created it

**Pull request merged**